package.json

```diff
@@ -53,9 +53,8 @@
     "jackspeak": "2.1.1"
   },
   "dependencies": {
-    "@storybook/channels": "^8.3.0",
-    "@storybook/telemetry": "^8.3.0",
-    "@storybook/types": "^8.3.0",
+    "@storybook/core-events": "^8.0.0 || ^8.1.0-0 || ^8.2.0-0 || ^8.3.0-0 || ^8.4.0-0 || ^8.5.0-0 || ^8.6.0-0",
+    "@storybook/telemetry": "^8.0.0 || ^8.1.0-0 || ^8.2.0-0 || ^8.3.0-0 || ^8.4.0-0 || ^8.5.0-0 || ^8.6.0-0",
```

Comment:
Just wondering about how this is going to resolve for this repository. Does the package manager pick the highest available option, or whichever is the first one in the pattern that's available? In the latter case I'm worried we might end up with v8.0.0 rather than a later minor version.